### ISSUE TITLE
Need a new npm namespace

### ISSUE THREAD

**Issue by kimdhamilton** [w3c-ccg/universal-wallet-interop-spec#72]
Per #71 we need to use a non-transmute npm namespace ASAP. I'm not sure whether it's been approved to publish CCG npm modules. We can use DCC's as a stopgap

**Comment by OR13** [comment]
This our our CD config, if someone can setup something similar we can easily replace `@transmute` with something else in this repo and have the same kind of convenient CD to NPM.

https://github.com/transmute-industries/linked-data/blob/main/.github/workflows/cd.yml

**Comment by Custard7** [comment]
Yes, would love to help out here @OR13 - @kimdhamilton let us know if CCG or DCC is a better destination for the CD to NPM flow.

**Comment by OR13** [comment]
also ping @wyc I believe you were trying to setup something like this before, I grant you permission to experiment on this repo / packages.

**Comment by kimdhamilton** [comment]
If the CCG namespace doesn't work, it sounds like DCC is acceptable? I'll wait a day for responses and proceed if there are no responses to the contrary

**Comment by kimdhamilton** [comment]
I think I need to update the names in the package.json files too

**Comment by OR13** [comment]
@kimdhamilton is this done? i think we can maybe close this.

**Comment by kimdhamilton** [comment]
Sorry for the delay; when I checked it didn’t seem to be publishing all the packages. I need to dig into this. Soon, I swear
